rectools/models/nn/transformers/hstu.py

```diff
@@ -0,0 +1,732 @@
+#  Copyright 2025 MTS (Mobile Telesystems)
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import typing as tp
+import warnings
+from typing import Dict
+
+import torch
+from torch import nn
+
+from ..item_net import (
+    CatFeaturesItemNet,
+    IdEmbeddingsItemNet,
+    ItemNetBase,
+    ItemNetConstructorBase,
+    SumOfEmbeddingsConstructor,
+)
+from .base import (
+    TrainerCallable,
+    TransformerDataPreparatorType,
+    TransformerLayersType,
+    TransformerLightningModule,
+    TransformerLightningModuleBase,
+    TransformerModelBase,
+    TransformerModelConfig,
+    ValMaskCallable,
+)
+from .data_preparator import InitKwargs, TransformerDataPreparatorBase
+from .negative_sampler import CatalogUniformSampler, TransformerNegativeSamplerBase
+from .net_blocks import LearnableInversePositionalEncoding, PositionalEncodingBase, TransformerLayersBase
+from .sasrec import SASRecDataPreparator
+from .similarity import DistanceSimilarityModule, SimilarityModuleBase
+from .torch_backbone import TransformerBackboneBase, TransformerTorchBackbone
+
+
+class RelativeAttention(torch.nn.Module):
```

Comment:
```suggestion
class RelativeAttentionBias(torch.nn.Module):
```
Since it's kind of the actual name and the full name for `rab`